scripts/generate-transactions.sh

```diff
@@ -4,4 +4,10 @@ rpc_endpoint=${1:-"http://127.0.0.1:8899"}
 
 echo "Using RPC Endpoint: $rpc_endpoint"
 
-solana airdrop --url $rpc_endpoint 10
\ No newline at end of file
+solana airdrop --url $rpc_endpoint 10
+
+solana transfer CSg2vQGbnwWdSyJpwK4i3qGfB6FebaV3xQTx4U1MbixN 1 --allow-unfunded-recipient --url $rpc_endpoint
```

Comment:
Solana CLI supports recipients as keypair paths, so what we do here is generate a new mock keypair under `fixtures`, then use that as the recipient.

```
mock_keypair_file="fixtures/mock-keypair.json"

solana-keygen new --no-bip39-passphrase -o $mock_keypair_file
solana transfer --allow-unfunded-recipient --url $rpc_endpoint $mock_keypair_file 1
```